src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat/SetTFMStep.cs

```diff
@@ -77,6 +77,6 @@ protected override Task<UpgradeStepInitializeResult> InitializeImplAsync(IUpgrad
             }
         }
 
-        protected override bool IsApplicableImpl(IUpgradeContext context) => context?.CurrentProject is not null;
+        protected override bool IsApplicableImpl(IUpgradeContext context) => context?.CurrentProject is not null && context.CurrentProject.IsSdk;
```

Comment:
Also, like the TryConvertStep I think this check should happen in InitializeImpl so that the step always shows up in the list of needed upgrade steps and then is skipped (with a warning) if the project isn't SDK-style when we get to this step.

Reply:
Actually, we don't need this at all since we're going to let the VB WPF apps fail.